LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaToolsTest/RegionImplementationTest.cs

```diff
@@ -24,8 +24,9 @@ public class RegionImplementationTest
             // in EU in standard parameters the expectation is that the reply arrive at same place.
             var expFreq = freq;
             var expDatr = datr;
-            Assert.Equal(RegionFactory.CreateEU868Region().GetDownstreamChannelFrequency(rxpk[0]), expFreq);
-            Assert.Equal(RegionFactory.CreateEU868Region().GetDownstreamDR(rxpk[0]), expDatr);
+            RegionManager.EU868.TryGetDownstreamChannelFrequency(rxpk[0], out double frequency);
```

Comment:
you have multiple of those. If you call a try, you should likely validate the return value of that (not going to add that comment all over :))

Reply:
Correct, I did that in the normal process, As those are in the tests, I assumed I would let it like this, but I will wrap those with an Assert.True